src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -27,7 +27,7 @@ public HashSet<int> Start(ApplicationData applicationData)
             _applicationData = applicationData;
             _gridViewDetails = new GridViewDetails
             {
-                // If we have an OutputMode, then we want to make them selectable. If we make them selectable,
+                // If OutputMode is Multiple, then we make items selectable. If we make them selectable,
                 // they have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
                 ListViewOffset = _applicationData.OutputMode != OutputModeOption.None ? 8 : 4
```

Comment:
don't you need to change this?

Reply:
The spacing is correct for both `Single` and `Multiple` modes now (we are now using `Terminal.Gui`'s `AllowsMultipleSelection` feature.

the comment is not quite right. I'll fix it.